paramak/parametric_components/poloidal_field_coil.py

```diff
@@ -20,31 +20,27 @@ def __init__(
         height,
         width,
         center_point,
-        workplane="XZ",
         rotation_angle=360,
-        solid=None,
         stp_filename="PoloidalFieldCoil.stp",
         color=None,
         azimuth_placement_angle=0,
-        points=None,
-        name=None,
+        name='pf_coil',
         material_tag="pf_coil_mat",
         cut=None,
-        hash_value=None,
```

Comment:
The alternative would be to collect these commonly unused arguments into a `**kwargs`, set the defaults in a dictionary below, and loop through the passed argument to modify those in the default dictionary, which is passed to `super.__init__()`

Something like (this could definitely be more pythonic, but you get the idea)
```python
default_dict = {...fill in}

for arg in kwargs:
    if arg in default_dict:
        default_dict[arg] = kwargs[arg]
```

Reply:
Thanks for the suggestion much appreciated, we can give this a go

Reply:
We have added kwargs to see how it looks :-)